assets/javascripts/kitten/karl/pages/recurring-donation-card/styles.js

```diff
@@ -0,0 +1,38 @@
+import styled from 'styled-components'
+import { pxToRem } from 'kitten'
+
+export const StyledRecurringDonation = styled.div` 
+  .k-RecurringDonation__thumbnailsTrio {
+    display: flex;
+    justify-content: space-between;
+    gap: ${pxToRem(5)};
+    margin-bottom: ${pxToRem(20)};
+  }
+
+  .k-RecurringDonation__thumbnail {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    border: var(--border);
+    width: ${pxToRem(93)};
+    height: ${pxToRem(89)};
+    box-sizing: border-box;
+    border-radius: var(--border-radius-m);
+    cursor: pointer;
+
+    &:hover,
+    &:focus {
+      border-color: var(--color-blue-300);
+    }
+
+    &:active {
+      border-color: var(--color-blue-500);
+      background-color: var(--color-blue-100);
+    }
+  }
```

Comment:
```suggestion
    &:hover,
    &:focus {
      border-color: var(--color-primary-300);
    }

    &:active {
      border-color: var(--color-primary-500);
      background-color: var(--color-primary-100);
    }
  }
```

Pourquoi ne pas avoir utilisé des `Button` dans ce cas ?

Reply:
Parce qu'aucun modifier ne correspond à ces styles !

Reply:
sur la maquette j'ai l'impression que c'est des Button avec un modifier `hydrogen` pour l'état normal et un modifier `lithium` pour l'état cliqué (https://kitten.vercel.app/?path=/story/molecules-buttons-button--all-modifiers)—ou même juste un Button par défaut avec la prop `active` à `true` pour l'élément sélectionné.